analytics/tests/analytics_test.cc

```diff
@@ -232,6 +232,16 @@ TEST_F(AnalyticsTest,
   InitiateOnDeviceConversionMeasurementWithEmailAddress("my_email");
 }
 
+TEST_F(AnalyticsTest,
+       TestInitiateOnDeviceConversionMeasurementWithPhoneNumber) {
+  // InitiateOnDeviceConversionMeasurementWithPhoneNumber is no-op on Android
+  AddExpectationApple(
+      "+[FIRAnalytics initiateOnDeviceConversionMeasurementWithPhoneNumber:]",
+      {"+15551234567"});
+
+  InitiateOnDeviceConversionMeasurementWithEmailAddress("+15551234567");
```

Comment:
Should be ***WithPhoneNumber

Do these tests work? :)

Reply:
Yeah, I'm not sure what the deal is with the tests, cause they probably aren't working correctly since they missed that.  I looked, but nothing obvious stood out.